src/main/java/org/pac4j/vertx/handler/impl/RequiresAuthenticationHandler.java

```diff
@@ -193,15 +193,15 @@ protected void authorise(final User user, final RoutingContext context) {
         );
     }
 
-    protected boolean useSession(final WebContext context, final List<Client> currentClients) {
```

Comment:
@millross is completely right: this is meant on purpose as an extension point which can be override.